players/installing-java/windows.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Installing Java on Windows
+description: A step by step guide on how to install Java on Windows.
+---
+
+# Installing Java on Windows
+
+This guide will walk you through installing Java 17 on Windows.
+
+The Minecraft Launcher comes with its own Java installation, so this section is only relevant if you want to use the Fabric `.jar` based installer, or if you want to use the Minecraft Server `.jar`.
+
+## 1. Verify if Java is already installed.
+
+To check if Java is already installed, you must first open the command prompt.
+
+You can do this by pressing <kbd>Win</kbd> + <kbd>R</kbd> and typing `cmd.exe` into the box that appears.
```

Comment:
More of a nitpick, but is `cmd.exe` used instead of just `cmd` for any reason? Also, we should direct users to Windows Terminal in Windows 11

Reply:
Windows terminal doesn't exist on early versions of windows, whilst CMD exists on nearly every version of windows.

Reply:
1. You didn't answer my first question

2. We already have to split everything up, having a note section saying to use windows terminal on 11

Reply:
? I did answer your question. There is zero need to make windows 11 users to use windows terminal when the old method works perfectly fine, and it's completely unnessecary to have another section to use a very specific piece of software for a single version of windows.

Reply:
The question you didn't answer is cmd.exe Vs cmd

Reply:
Ah, it's clearer to users. That's it.